android/engine/src/main/java/org/smartregister/fhircore/engine/task/FhirCarePlanGenerator.kt

```diff
@@ -99,6 +100,14 @@ constructor(
         .search<CarePlan> {
           filter(CarePlan.INSTANTIATES_CANONICAL, { value = planDefinition.referenceValue() })
           filter(CarePlan.SUBJECT, { value = subject.referenceValue() })
+          filter(
+            CarePlan.STATUS,
+            { value = of(CarePlanStatus.DRAFT.toCode()) },
+            { value = of(CarePlanStatus.ACTIVE.toCode()) },
+            { value = of(CarePlanStatus.ONHOLD.toCode()) },
+            { value = of(CarePlanStatus.ENTEREDINERROR.toCode()) },
```

Comment:
@aurangzaibumer I think we can remove this. By the time the CarePlan us marked as entered in error it means the it should be closed